sdk/core/azure-core/inc/azure/core/etag.hpp

```diff
@@ -135,7 +134,7 @@ class ETag final {
    */
   bool HasValue() const { return m_value.HasValue(); }
 
-  /*
+  /**
```

Comment:
Nice! How did you find this issue?

Reply:
By luck while trying to see how we were documenting the `ToString` methods.  I was expecting to find this in the eTag docs and I didn't see it.

Reply:
Maybe there is some regex we can run to detect this issue, if it is happening in more places.